packages/app-store/components.tsx

```diff
@@ -29,7 +30,7 @@ export const InstallAppButton = (
 ) => {
   const { status } = useSession();
   const { t } = useLocale();
-  const appName = props.type.replace("_", "") as keyof typeof InstallAppButtonMap;
+  const appName = props.type.replaceAll("_", "") as keyof typeof InstallAppButtonMap;
```

Comment:
Same as before, the type can now be `{name}_other_calendar` so making sure we replace every underscore for empty string.